pkg/promql/engine.go

```diff
@@ -69,7 +71,7 @@ type Querier interface {
 }
 
 const (
-	namespace            = "prometheus"
+	namespace            = util.PromNamespace
```

Comment:
Breaking change but necessary. @cevian

Reply:
We do not want our PromQL engine stats to have names that match exactly with that of Prometheus's PromQL engine.

Reply:
Please make a note on the release notes draft

Reply:
Updated the release. I hope the sentence is not too long.